README.md

```diff
@@ -1,10 +1,44 @@
 # xsamtools
-Lightly-modified versions of [htslib](https://github.com/samtools/htslib)
-and [bcftools](https://github.com/samtools/bcftools), available through pypi packaging, with capability to merge VCF
-streams.
+xsamtools makes the [samtools](https://samtools.github.io/) tooling from [htslib](https://github.com/samtools/htslib)
+and [bcftools](https://github.com/samtools/bcftools) available through pypi packaging. These tools have been lightly
+modified to allow merges on VCF streams without an index.
 
-# installation
+# Installation
 ```
 pip install xsamtools
 ```
-A C toolchain is required to install xsamtools.
+Installation requires a C toolchain.
+
+# Usage
+
+After succesful installation, the following executables are available:
```

Comment:
What happens if `htsfile` is already installed on the system, is it overwritten?

Reply:
If you're installing into a virtual environment the binaries will end up there. For system installs I suppose it's a possibility.